parsel/selector.py

```diff
@@ -223,6 +237,44 @@ def __getstate__(self):
     def _get_root(self, text, base_url=None):
         return create_root_node(text, self._parser, base_url=base_url)
 
+    def jmespath(self, query, **kwargs):
+        """
+        Call the ``.jmespath()`` method for each element in this list and return
+        their results flattened as another :class:`SelectorList`.
+
+        ``query`` is the same argument as the one in :meth:`Selector.jmespath`
+
+        Requires : jmespath(https://github.com/jmespath/jmespath)
```

Comment:
This description seems copied from SelectorList.jmespath. This description should instead be more detailed, in line with the `xpath` description below.

It should probably include a link to http://jmespath.org/ as a documentation reference for the JMESPath syntax. And mentioning that `jmespath` is required should not be necessary if we make `jmespath` part of the Parsel requirements; if we don’t, then the import of `jmespath` cannot be at the beginning of the file.